test/sql/drop_sql_test.cpp

```diff
@@ -0,0 +1,76 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// drop_sql_test.cpp
+//
+// Identification: test/sql/drop_sql_test.cpp
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#include <memory>
+
+#include "catalog/catalog.h"
+#include "common/harness.h"
+#include "executor/create_executor.h"
+#include "optimizer/simple_optimizer.h"
+#include "planner/create_plan.h"
+
+#include "sql/sql_tests_util.h"
+
+namespace peloton {
+namespace test {
+
+class DropSQLTests : public PelotonTest {};
+
+TEST_F(DropSQLTests, DropSQLTest) {
+  catalog::Catalog::GetInstance()->CreateDatabase(DEFAULT_DB_NAME, nullptr);
+
+  // Create a table first
+  SQLTestsUtil::ExecuteSQLQuery("CREATE TABLE test(a INT PRIMARY KEY, b INT);");
+
+  // Check the table in catalog
+  EXPECT_EQ(catalog::Catalog::GetInstance()
+                ->GetDatabaseWithName(DEFAULT_DB_NAME)
+                ->GetTableCount(),
```

Comment:
You should look for the actual Table instead of just getting the count in case we start adding internal tables.